uaclient/util.py

```diff
@@ -3,8 +3,8 @@
 import logging
 import os
 import re
-import six
 import subprocess
+import urllib
```

Comment:
in python3.7 (disco) maybe even 3.4 and 3.5 urllib doesn't surface submodule names directly on the urllib module so It doesn't have to load all submodules I believe.
As a result we need 
```
from urllib import request

and below change the call sites to 

request.Request
&& 
request.urlopen